base/compiler/abstractinterpretation.jl

```diff
@@ -670,11 +670,11 @@ function abstract_call(@nospecialize(f), fargs::Union{Tuple{},Vector{Any}}, argt
         rty = abstract_call(<:, fargs, argtypes, vtypes, sv)
         return rty
     elseif length(argtypes) == 2 && isa(argtypes[2], Const) && isa(argtypes[2].val, SimpleVector) && istopfunction(tm, f, :length)
-        # mark length(::SimpleVector) as @pure
+        # mark length(::SimpleVector) as @unsafe_pure
         return Const(length(argtypes[2].val))
     elseif length(argtypes) == 3 && isa(argtypes[2], Const) && isa(argtypes[3], Const) &&
             isa(argtypes[2].val, SimpleVector) && isa(argtypes[3].val, Int) && istopfunction(tm, f, :getindex)
-        # mark getindex(::SimpleVector, i::Int) as @pure
+        # mark getindex(::SimpleVector, i::Int) as @unsafe_pure
```

Comment:
These two should probably say "as pure", since it's only indirectly connected to the `@pure` macro